tools/png2obj/bleveler.py

```diff
@@ -0,0 +1,560 @@
+# -*- coding: utf-8 -*-
```

Comment:
This one duplicates a lot of code which is part of normal CLI tool. Maintaining one would imply to go around the other and apply changes. If there's no easy way so far to create a Blender plugin with importable external modules, lets leave this out for now.